tests/python/gaia-ui-tests/gaiatest/tests/graphics/test_setup_and_send_receive_imap_email.py

```diff
@@ -0,0 +1,112 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette import SkipTest
+from marionette_driver import Wait
+
+from gaiatest.gaia_graphics_test import GaiaImageCompareTestCase
+from gaiatest.apps.email.app import Email
+from gaiatest.apps.email.app import SetupEmail
+
+class TestSetupAndSendIMAPEmail(GaiaImageCompareTestCase):
+
+    def setUp(self):
+        try:
+            self.testvars['email']['imap']
+            self.testvars['email']['smtp']
+        except KeyError:
+            raise SkipTest('account details not present in test variables')
+
+        #GaiaImageCompareTestCase.setUp(self)
+        GaiaImageCompareTestCase.setUp(self)
+        self.connect_to_local_area_network()
+
+        self.email = Email(self.marionette)
+        self.email.launch()
+
+    def test_setup_and_send_imap_email(self):
+        """
+        https://moztrap.mozilla.org/manage/case/6113/
+        https://moztrap.mozilla.org/manage/case/6114/
```

Comment:
You may have copied this style from elsewhere but it's not a good docstring. The first line should be a summary, and any additional lines should contain details. See http://legacy.python.org/dev/peps/pep-0257/#multi-line-docstrings